src/global/utils.ts

```diff
@@ -0,0 +1,36 @@
+import $rdf, { Fetcher, IndexedFormula, NamedNode, sym } from "rdflib"
```

Comment:
I generally try to avoid naming things `util` if I can, since it doesn't really say anything about the contents. Perhaps this file could be called `metadata.ts`, `getMetadata.ts`, `getPodMetadata.ts` or something like that, since all these functions seem to do something related to that?

(Two hard things in software engineering? :P)